src/qt/locale/bitcoin_pl.ts

```diff
@@ -6,12 +6,12 @@
     <message>
         <location filename="../forms/aboutdialog.ui" line="+14"/>
         <source>About Gridcoin</source>
-        <translation type="unfinished"></translation>
+        <translation>O oprogramowaniu GridCoin</translation>
     </message>
     <message>
         <location line="+39"/>
         <source>&lt;b&gt;Gridcoin&lt;/b&gt; </source>
-        <translation type="unfinished"></translation>
+        <translation>Gridcoin </translation>
```

Comment:
I think you want to retain the format (bold?) here, so the translation will be the same as the source.